tslint.json

```diff
@@ -1,14 +1,49 @@
 {
-    "rules": {
-        "no-any": false,
-        "no-empty": true,
-        "no-empty-interface": true,
-        "no-duplicate-imports": true,
-        "no-duplicate-variable": true,
-        "no-unused-variable": true,
-        "triple-equals": true,
-        "prefer-const": true,
-        "arrow-return-shorthand": true,
-        "no-invalid-template-strings": true
-    }
-}
\ No newline at end of file
+  "rules": {
+    "no-any": false,
+    "no-empty": true,
+    "no-empty-interface": true,
+    "no-duplicate-imports": true,
+    "no-duplicate-variable": true,
+    "triple-equals": true,
+    "prefer-const": true,
+    "no-invalid-template-strings": true,
+    "arrow-return-shorthand": [true, "multiline"],
+    "no-unnecessary-callback-wrapper": true,
+    "typedef": [
+      true,
+      "call-signature",
+      "parameter",
+      "arrow-call-signature",
+      "arrow-parameter",
+      "property-declaration"
+    ],
+
+    "ordered-imports": true,
+    "object-literal-sort-keys": [
+      true,
+      "ignore-blank-lines",
+      "ignore-case",
+      "locale-compare",
+      "match-declaration-order",
```

Comment:
My suggestion is that if we want to stick strictly to alphabetical order (as declared above), we'd rather remove this line.

Reply:
Finally - I deleted "match-declaration-order" cuz was causing an error, thanks for your notice anyway.